fe/fe-core/src/main/java/com/starrocks/qe/ShowExecutor.java

```diff
@@ -489,14 +492,27 @@ private void handleShowMigrations() throws AnalysisException {
     }
 
     // Show databases statement
-    private void handleShowDb() throws AnalysisException {
+    private void handleShowDb() throws AnalysisException, DdlException {
```

Comment:
why must throw DdlException?

Reply:
> why must throw DdlException?

cause this func calls listDbNames to get databases names which would throw DdlException
default List<String> listDbNames() throws DdlException {
     return Lists.newArrayList();
}

Reply:
Why list db names throw Data **“Definition”** Language Exception, I think it's illegal

Reply:
> 

it would be fixed by another pr, i need to wait for it got merged.